src/test/java/study/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
```

Comment:
```suggestion
class StringAddCalculatorTest {
    @Test
    void splitAndSum_null_또는_빈문자() {
```
junit5부터는 테스트 클래스에 public 접근제어자가 제거되어도 됩니다 :)

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods